src/features/components/templates/navigation/template/desktop-menu/styles.scss

```diff
@@ -45,6 +45,13 @@
         animation-duration: 250ms;
         animation-timing-function: ease;
     }
+    &_scrollable_menu {
+        @media screen and (max-height: 700px) {
+            block-size: 100vh;
+            padding-block-end: 10rem;
+            overflow: scroll;
```

Comment:
Hi @mohsen-deriv , can we go with overflow: auto;  ?

Reply:
thanks, updated :pray: